.github/workflows/bokeh-ci.yml

```diff
@@ -349,3 +354,120 @@ jobs:
           BOKEH_DOCKER_INTERACTIVE: 0
         run: |
           scripts/docker/docker_run.sh $IMAGE_TAG
+
+  bokehjs-test:
```

Comment:
What's the rationale for merging bokehjs' workflow with bokeh's?

Reply:
My line of thinking was the following: If the only thing that the `bokehjs` CI is doing is to run the bokehjs tests and if this part should also run conditionally (only if changes were made in this folder), then add it to the "main" CI, so that the conditionals a) share the same logic and b) can be found in one place.

Reply:
@mattpap FWIW It is common to need to restart tests in both Bokeh and BokehJS these days, and it is a minor  
 but *persistent* annoyance to not be able to do that all in one place. 

I'll turn the question around: what's the rationale to *not* have a combined workflow? Is the goal that bokehjs-only changes don't trigger the full test suite? That's reasonable and doable -- we need to split out the selenium integration tests (e.g. for image export) from the python unit tests.